rir/src/compiler/translations/pir_2_rir/pir_2_rir.cpp

```diff
@@ -782,7 +789,9 @@ size_t Pir2Rir::compileCode(Context& ctx, Code* code) {
 
             case Tag::LdVar: {
                 auto ldvar = LdVar::Cast(instr);
-                cs << BC::ldvarNoForce(ldvar->varName);
+                cs << (ldvar->fusedWithForce
```

Comment:
If when coming from pir forces are always explicit, do we need to lower to `ldVar` at all?

Reply:
I guess there can be some reordering of instructions that can move the load and force apart